yang/ietf-l2vpn-ntw.yang

```diff
@@ -1045,36 +1045,87 @@ module ietf-l2vpn-ntw {
                           description
                             "p2mp-replication  TBC";
                         }
-                        leaf arp-proxy {
-                          type boolean;
-                          default "false";
-                          description
-                            "Enables (when set to 'true') or disables (when set
-                             to 'false') ARP proxy.";
-                        }
-                        leaf arp-suppression {
-                          if-feature "vpn-common:ipv4";
-                          type boolean;
-                          default "false";
-                          description
-                            "Enables (when set to 'true') or disables (when set
-                             to 'false') ARP suppression.";
-                        }
-                        leaf nd-proxy {
-                          if-feature "vpn-common:ipv6";
-                          type boolean;
-                          default "false";
+                        container arp-proxy {
+                          leaf arp-proxy-enable {
+                            type boolean;
+                            default "false";
+                            description
+                              "Enables (when set to 'true') or disables (when set
+                               to 'false') ARP proxy.";
+                          }
+                          leaf arp-suppression {
+                            if-feature "vpn-common:ipv4";
+                            type boolean;
+                            default "false";
+                            description
+                              "Enables (when set to 'true') or disables (when set
+                               to 'false') ARP suppression.";
+                            reference
+                              "draft-ietf-bess-evpn-proxy-arp-nd-13
+                              RFC 7432: BGP MPLS-Based Ethernet VPN";
+                          }
+                          leaf ip-mobility-threshold {
```

Comment:
Where is this defined?

Reply:
Sorry the delay. It Is defined https://datatracker.ietf.org/doc/html/draft-ietf-bess-evpn-proxy-arp-nd-14#section-3.7